window.guardian.d.ts

```diff
@@ -44,6 +41,7 @@ declare global {
         };
         GoogleAnalyticsObject: string;
         ga: UniversalAnalytics.ga;
+        guardianPolyfilledImport: (url: string) => Promise<any>; // can't be nested beyond top level
```

Comment:
It'd be ideal to pop a detailed comment for what this for above this as it'll then appear in the type intellisense.

Reply:
Hey yeah that's a great shout, thanks. I've added something, does it match what you were thinking?